ControlMeasurements/Data/WaterContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ControlMeasurements.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ControlMeasurements.Data
+{
+    public class WaterContext : DbContext
```

Comment:
Ta aplikacja ma służyć tylko do pomiarów wody? Czy inne liczniki też? Jeśli inne liczniki też, to lepsza by była jakaś bardziej ogólna nazwa dla contextu (reprezentuje Twoja baze danych), np. "MeasurementsContext". Ale na razie może być :)

Reply:
Poprawione :)